H2TauTau/python/heppy/ntuple/ntuple_variables.py

```diff
@@ -9,7 +9,7 @@
     lumi = v(lambda x: x.lumi, int),
     event = v(lambda x: x.eventId, int, 'l'),
     n_up = v(lambda x: getattr(x, 'NUP', default), int),
-    n_pu = v(lambda x: x.nPU if x.nPU is not None else default),
+    n_pu = v(lambda x: getattr(x, 'nPU', default) if getattr(x, 'nPU', default) is not None else default, int),
```

Comment:
add comments for all these changes (at least write that you cover specificities of the embedded samples )